docs/site_specific_config/host_injections.md

```diff
@@ -0,0 +1,43 @@
+# How to configure EESSI
+
+## The `host_injections` variant symlink
+
+While the EESSI software stack aims to work 'out of the box', one cannot avoid some degree of site-specific configuration for certain functionality and/or performance tuning. For example, GPU drivers may be in different locations on different systems. Also, a site might want to tune the OpenMPI installation that comes with EESSI for optimal performance on the local hardware.
```

Comment:
The first sentence seems to be counter-intuitive to what EESSI tries to achieve. It is more reasonable to say that due to the copyrights of the software or freedom of configuration, the site specific configuration with host injection is enabled in EESSI.

Reply:
> The first sentence seems to be counter-intuitive to what EESSI tries to achieve

Not really. The primary goal is to offer the same, optimized software stack everywhere. Sure, it is a benefit if it's easy to deploy EESSI, as that will increase uptake. And where possible, we abstract from host-specific differences. But there _are_ differences between hosts that we simply have to deal with through configuration, the prime example being the location of the GPU drivers for GPU functionality.

I'll think about the phrasing though, maybe I shouldn't start with the statement that we want things to work out of the box - that might simply be too optimistic.

Reply:
Ok, made an additional (very short) section explaining when/why configuration i needed. Also, removed the phrasing which states that we aim to work 'out-of-the-box', that's simply not feasible, so no point in mentioning that.